reviews/models/mixins.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from transitions import Machine
+
+from django.db import models
+
+from reviews.utils import AttrProxy
+from reviews.workflow import States
+from reviews.workflow import Workflows
+from reviews.workflow import TRANSITIONS
+
+
+class ReviewProviderSettings(models.Model):
```

Comment:
I think I prefer the mixin here, I may have done a bad job of describing what I was thinking.

The mixin still exists with `reviews_` prefixes on all the fields

but you end up with something like:

```python
class ReviewsSettingsNames:
     # getters and setters that forward "name" -> "model_instance.reviews_name"

    def __init__(self, model_instance):
         ...

class YourCoolMixin:

   @property
    def review_settings:
         return ReviewSettings(self)

```